mm2src/mm2_test_helpers/dummy_files/iris_nimda_history.json

```diff
@@ -0,0 +1,391 @@
+        "tx_hash": "D83D8143CAD8C225F3BC24C6DCA324DB2F63C97A21AE2C030328266E9415E50A",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [
+            "iaa1eg0qgaz73jsvvrvvtzq4x823hmz8qapldd0x4z"
+        ],
+        "total_amount": "0.0001",
+        "spent_by_me": "0.0001",
+        "received_by_me": "0",
+        "my_balance_change": "-0.0001",
+        "block_height": 6148307,
+        "timestamp": 1669798865,
+        "fee_details": {
+            "type": "Tendermint",
+            "coin": "IRIS-NIMDA",
+            "amount": "0.025763",
+            "gas_limit": 100000
+        },
+        "coin": "IRIS-NIMDA",
+        "internal_id": "3643343243423346353232433844414333343138443338440000000000000000",
+        "transaction_type": {
+            "TokenTransfer": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
+        }
+    },
+    {
+        "tx_hex": "0a2a696161316530727838376d646a37397a656a65777563346a6737716c3975643232383667327573386632122a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301a0e0a036e696d120731303030303030",
+        "tx_hash": "7A43C30967D30B12B749111F217A670C0B47B7C5F272BCD82C442C1554351CB7",
+        "from": [
+            "iaa1e0rx87mdj79zejewuc4jg7ql9ud2286g2us8f2"
+        ],
+        "to": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "total_amount": "2",
+        "spent_by_me": "0",
+        "received_by_me": "2",
+        "my_balance_change": "2",
+        "block_height": 6148055,
+        "timestamp": 1669797598,
+        "fee_details": {
+            "type": "Tendermint",
+            "coin": "IRIS-NIMDA",
+            "amount": "0.022114",
+            "gas_limit": 100000
+        },
+        "coin": "IRIS-NIMDA",
+        "internal_id": "4631313139343742323142303344373639303343333441370000000000000000",
+        "transaction_type": {
+            "TokenTransfer": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
+        }
+    },
+    {
+        "tx_hex": "0a2a696161316530727838376d646a37397a656a65777563346a6737716c3975643232383667327573386632122a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a7261301a0e0a036e696d1207313030303030301a100a05756e79616e120731303030303030",
+        "tx_hash": "3057F075161ECD149F4F5A629F852D5E7E57411DECE7AD35CC7D11E7EB06E0BF",
+        "from": [
+            "iaa1e0rx87mdj79zejewuc4jg7ql9ud2286g2us8f2"
+        ],
+        "to": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "total_amount": "2",
+        "spent_by_me": "0",
+        "received_by_me": "2",
+        "my_balance_change": "2",
+        "block_height": 6148039,
+        "timestamp": 1669797517,
+        "fee_details": {
+            "type": "Tendermint",
+            "coin": "IRIS-NIMDA",
```

Comment:
I'm confused a bit. `IRIS-NIMDA` is a token over `IRIS-TEST` platform coin. I guess the fee should be paid in `IRIS-TEST`, isn't it?